assets/chezmoi.io/mkdocs.yml

```diff
@@ -244,6 +245,9 @@ nav:
       - pass: reference/templates/pass-functions/pass.md
       - passFields: reference/templates/pass-functions/passFields.md
       - passRaw: reference/templates/pass-functions/passRaw.md
+    - passhole function:
+      - reference/templates/passhole-funtions/index.md
```

Comment:
There's a typo here: `funtions` should be `functions`. Was the website tested as part of this PR? See https://www.chezmoi.io/developer/website/.